src/objective-c/ProtoRPC/ProtoService.m

```diff
@@ -65,6 +65,8 @@ - (instancetype)initWithHost:(NSString *)host
   return self;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wincompatible-pointer-types"
```

Comment:
Ah, then let's just create a `GRPCProtoMethod` in line 74, no? :) Better than silencing a warning we want.